### DISCUSSION TITLE
optionally display line-numbers

### DISCUSSION THREAD

**Discussion by Yellow-Subroutine** [gsantner/markor#2011] [Ideas] [answered]
Hi,

it sometimes would be useful if one could see the line-numbers of markdown documents.

E.g. I update my some of my files with scripts and it would be nice to be able to easily say (to a script) "update the content between lines x and y.

Of course the display of line-numbers should be optional.

**Comment by gsantner** [comment, marked as answer]
please search issues. it has been asked a few times already.
